notebooks/demo-train-with-faq.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<p>In the interest of inclusive language, let's use something like <em>verification</em> instead of <em>sanity check</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Good point :)</p>